templates/common/_base/files/configure-ovs-network.yaml

```diff
@@ -87,8 +88,54 @@ contents:
     nmcli conn up ovs-if-phys0
 
     if ! nmcli connection show ovs-if-br-ex &> /dev/null; then
-      nmcli c add type ovs-interface slave-type ovs-port conn.interface br-ex master ovs-port-br-ex con-name \
-        ovs-if-br-ex 802-3-ethernet.mtu ${iface_mtu} 802-3-ethernet.cloned-mac-address ${iface_mac}
+      if nmcli --fields ipv4.method,ipv6.method conn show $old_conn | grep manual; then
+        echo "Static IP addressing detected on default gateway connection: ${old_conn}"
+        # find and copy the old connection to get the address settings
+        old_conn_file=$(egrep -l --include=*.nmconnection $old_conn ${NM_CONN_PATH}/*)
+        if [ -z "$old_conn_file" ]; then
+          echo "ERROR: unable to find NM configuration file for conn: ${old_conn}"
+          exit 1
+        fi
+        echo "old connection file found at: ${old_conn_file}"
+        new_conn_file=${NM_CONN_PATH}/ovs-if-br-ex.nmconnection
+        if [ -f "$new_conn_file" ]; then
+          echo "WARN: existing br-ex interface file found: $new_conn_file, which is not loaded in NetworkManager...overwriting"
+        fi
+        cp -f ${old_conn_file} ${new_conn_file}
+        ovs_port_conn=$(nmcli --fields connection.uuid conn show ovs-port-br-ex | awk '{print $2}')
+        br_iface_uuid=$(cat /proc/sys/kernel/random/uuid)
+        # modify file to work with OVS and have unique settings
+        sed -i '/^\[connection\]$/,/^\[/ s/^uuid=.*$/uuid='"$br_iface_uuid"'/' ${new_conn_file}
```

Comment:
for the sake of readability and maintainability, can you group the `sed` changes by the section being modified? The connection ones ought to be together, the ethernet ones got to be clubbed etc.